### ISSUE TITLE
Stop ingest from using too much RAM

### ISSUE THREAD

**Issue by lolpro11** [CatenaryMaps/catenary-backend#47]
Maybe don't db dump all at once?

**Comment by kylerchin** [comment]
I agree, this seems to be a major limitation with large feeds such as switzerland, UK, Deutchland

Maybe we perform feed splitting?

**Comment by kylerchin** [comment]
I don't know how to approach the problem
